Create postBuild (#2)

darwintree opened the pull request:
doc: fix dependency for my builder

benjaoming commented on postBuild:
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+set -e
```
```suggestion
# This script is invoked by the mybinder.org service which can detect,
# display and execute any .ipynb Jupyter Notebook in the repository.
set -e
```

benjaoming commented on postBuild:
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+set -e
+pip install -r docs/requirements.txt
```
```suggestion
# We reuse the requirements from the documentation project
pip install -r docs/requirements.txt
```

benjaoming reviewed (commented):
Adding some comments for users of the repository to understand what's going on.

Another note: Do you think that it's problematic that the requirements also include the entire Sphinx stack? Should we split up the requirements?

I have a slight preference for _not_ "disturbing" the Sphinx setup.

darwintree commented:
> Adding some comments for users of the repository to understand what's going on.
> 
> Another note: Do you think that it's problematic that the requirements also include the entire Sphinx stack? Should we split up the requirements?
> 
> I have a slight preference for _not_ "disturbing" the Sphinx setup.

Yes, I think it would be better if we split the dependencies in `requirements.txt`, but as this is an example project, this might confuse readers
